db/db.go

```diff
@@ -134,6 +140,50 @@ func (db *DB) Close() error {
 	return db.sqlite3conn.Close()
 }
 
+// Destroy drops all the tables from the database. This is useful for resetting
+// an in-memory database when necessary.
+func (db *DB) Destroy() error {
```

Comment:
The problem is that you don't necessarily know when you want to do that. The stdlib's connection pooling mechanism may open or close connections at will, and you don't want to delete the entire database when one connection in the pool is closed, because the others are still valid.